### PR TITLE
Link video

### PR DESCRIPTION
**Pull request by janhohenheim**
[Rendered](https://github.com/TheBevyFlock/bevy_quickstart/blob/video/README.md)

### CODE REVIEW

**Comment by janhohenheim** [comment]
Requesting @christopherbiscardi's review here as well :)

**Comment by ChristopherBiscardi** [comment]
lgtm

here's the original thumbnail if you want to avoid hitlinking: 

opengraph-jam copy

**Comment by janhohenheim** [comment]
Thanks! I added a little play button to it while I was on it to make it more obvious that people should click on it :)

File: README.md [hunk 1]
```diff
@@ -6,6 +6,9 @@ This template is a great way to get started on a new [Bevy](https://bevyengine.o
 Start with a [basic project structure](#write-your-game) and [CI / CD](#release-your-game) that can deploy to [itch.io](https://itch.io).
 You can [try this template in your web browser!](https://the-bevy-flock.itch.io/bevy-quickstart)
 
+Don't want to read through the whole README? [@ChristopherBiscardi](https://github.com/ChristopherBiscardi) made a video tutorial explaining how to use the template from start to finish.
+[![A video tutorial for bevy_quickstart](./docs/thumbnail.png)](https://www.youtube.com/watch?v=ESBRyXClaYc)
```
Comment (benfrankel) [inline README.md:10]: The video may become outdated over time, but for now this is good.

**Review by benfrankel — APPROVED** [review]

**Comment by janhohenheim** [comment]
Making it collapsible would be a bit of a pity imo. I made it smaller instead.

**Comment by benfrankel** [comment]
> Making it collapsible would be a bit of a pity imo. I made it smaller instead.

Okay that's fine, it was just huge in the render.

File: README.md [hunk 2]
```diff
@@ -6,6 +6,10 @@ This template is a great way to get started on a new [Bevy](https://bevyengine.o
 Start with a [basic project structure](#write-your-game) and [CI / CD](#release-your-game) that can deploy to [itch.io](https://itch.io).
 You can [try this template in your web browser!](https://the-bevy-flock.itch.io/bevy-quickstart)
 
+Don't want to read through the whole README? [@ChristopherBiscardi](https://github.com/ChristopherBiscardi) made a video explaining how to use the template from start to finish.
```
Comment (benfrankel) [inline README.md:9]:
```suggestion
[@ChristopherBiscardi](https://github.com/ChristopherBiscardi) made a helpful video on how to use this template from start to finish:
```

**Review by benfrankel — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (benfrankel) [reply README.md:9]: If not this exact wording, my main issue with this line is that the final word wraps to a new line.

Comment (janhohenheim) [reply README.md:9]: image

That's on your specific screen settings. I'll change the wording a bit more, but I don't want to spend more time optimizing that.

Comment (janhohenheim) [reply README.md]: [`9e359d5` (#174)](https://github.com/TheBevyFlock/bevy_quickstart/pull/174/commits/9e359d5093199a49a508caead030b4c6711792d7)

**Review by benfrankel — APPROVED** [review]
I don't like that we're suggesting not to read the README, in the README. But it's fine.

**Pull request merged**